build.sbt

```
@@ -41,7 +41,7 @@ resolvers ++= Seq(
 )
 
 libraryDependencies ++= Seq(
-  "org.scalatest" %% "scalatest" % "3.0.8",
+  "org.scalatest" %% "scalatest" % "3.2.1",
```

```suggestion
  "org.scalatest" %% "scalatest" % "3.2.0",
```

I don't know if it's relevant here but we had to roll back to `3.2.0` on FIRRTL: https://github.com/freechipsproject/firrtl/pull/1845

---

No probs, I'll revert the commit to upgrade to 3.2.1.

---

Done